vcpkg.json

```diff
@@ -14,15 +14,18 @@
     "coin-or-cbc",
     {
       "name": "boost-mpi",
-      "version>=": "1.81.0"
+      "version>=": "1.81.0",
+      "platform": "windows"
     },
     {
       "name": "boost-program-options",
-      "version>=": "1.81.0"
+      "version>=": "1.81.0",
+      "platform": "windows"
```

Comment:
why ?

Reply:
if we pull boost-mpi from vcpkg, it'll also install mpi --> against the Pr objective